src/components/scenes/WalletListScene.js

```diff
@@ -101,21 +101,23 @@ class WalletListComponent extends React.PureComponent<Props, State> {
           onPress={this.props.toggleAccountBalanceVisibility}
           exchangeRates={this.props.exchangeRates}
         />
-        <View /* header stack */>
-          {/* <SettingsHeaderRow icon={<Fontello name="wallet-1" size={theme.rem(1.25)} color={theme.icon} />} text={s.strings.fragment_wallets_header} /> */}
-          <CrossFade activeKey={sorting ? 'doneButton' : 'defaultButtons'}>
+        <View style={styles.headerContainer}>
+          <EdgeText style={styles.headerText}>{s.strings.title_wallets}</EdgeText>
+          {!sorting && (
             <View key="defaultButtons" style={styles.headerButtonsContainer}>
               <TouchableOpacity style={styles.addButton} onPress={Actions[Constants.CREATE_WALLET_SELECT_CRYPTO]}>
-                <Ionicon name="md-add" size={theme.rem(1.25)} color={theme.iconTappable} />
+                <Ionicon name="md-add" size={theme.rem(1.5)} color={theme.iconTappable} />
               </TouchableOpacity>
               <TouchableOpacity onPress={this.handleSort}>
-                <Fontello name="sort" size={theme.rem(1.25)} color={theme.iconTappable} />
+                <Fontello name="sort" size={theme.rem(1.5)} color={theme.iconTappable} />
               </TouchableOpacity>
             </View>
+          )}
+          {sorting && (
             <TouchableOpacity key="doneButton" style={styles.headerButtonsContainer} onPress={this.disableSorting}>
               <EdgeText>{s.strings.string_done_cap}</EdgeText>
             </TouchableOpacity>
-          </CrossFade>
+          )}
```

Comment:
Why was cross fade removed?

Reply:
Messes with the design because of it's restrictive style